src/pkg/selectors/sharepoint.go

```diff
@@ -0,0 +1,425 @@
+package selectors
```

Comment:
This file was generated by
1. copy-pasting /selectors/onedrive.go
2. replacing all "onedrive" -> "sharepoint" and "user" -> "site"
3. Removing any vestigial parts like the onedrive filters.

Any code that couldn't compile due to missing dependencies is commented out.